detekt-config.yml

```diff
@@ -255,7 +255,7 @@ style:
     active: true
   UnderscoresInNumericLiterals:
     active: false
-    acceptableDecimalLength: 3
+    acceptableLength: 3
```

Comment:
I assume this update is only because Detekt made that change in https://github.com/detekt/detekt/commit/f9205046fb3cd3e75b63a77149b94324b44c71fe

Good catch!